templates/admin/transaction/search_by_company_number.html.tx

```diff
@@ -5,6 +5,8 @@
     % include includes::transactions {title => $title ~ ' ' ~ $search}
 
     % if !$transactions {
-        <p><strong>No results found.</strong></p>
+        <div id="no-results" role="status">
+                <p><strong>No results found.</strong></p>
```

Comment:
Isn't the indentation too large for the nested <p> tag? 8 spaces instead of 4?

Reply:
Good spot Hywel. I'll fix that. 👍